### DISCUSSION TITLE
Unlocalisable phrases due to Crowdin configuration

### DISCUSSION THREAD

**Discussion by bloodwiing** [anyproto/contributors#161] [Localization]
This is a repost from Crowdin. The discussion post just did not seem to be seen, so I thought to try my luck here.

But, essentially, there are certain Strings that collide and require a word to mean a noun and verb at the same time, which isn't possible in most languages apart from English.

The original post is:
https://crowdin.com/project/anytype-desktop/discussions/52

Can there be any comment on this?

**Comment by fuksman** [comment]
Hey @bloodwiing, thanks for the report! Sorry we missed it on Crowdin, I've replied you there.

**Reply by bloodwiing** [reply]
I sent a reply on crowdin
